README.md

```diff
@@ -21,3 +26,56 @@ Then you can run it on every file via
 `ormolu --mode inplace $(git ls-files '*.hs')`
 
 You can see full documentation on the [Ormolu repo](https://github.com/tweag/ormolu#usage).
+
+## Good Practice Recommendations
+
+Here are some good practice tips for git and this project.
+
+* Whenever a new feature is being worked on, create a new branch on your forked repo
+* When a feature is ready to be merged into the bot, make a pull request from the feature branch to the main repo
+* Before making a pull request, make sure your branch is up to date with main (and that it compiles and complies with `ormolu`) so that it can be merged without hassle
```

Comment:
Bonus point to add: something about communicating with the maintainers (Discord is good)